### ISSUE TITLE
✨ `gradient-text` semantic + `gradient()` shorthand

### ISSUE THREAD

**Issue by 1aron** [master-co/css#167]
### Description

```html
<div class="bg:text text-fill-color:transparent bg:linear-gradient(45deg,#f3ec78,#af4261)">Hello World</div>
```
Now simplify it:
```html
<div class="gradient-text gradient(45deg,#f3ec78,#af4261)">Hello World</div>
```

**Comment by Awilum** [comment]
so, functions and custom functions already implemented ?)

**Comment by 1aron** [comment]
@Awilum #46 not yet. 2.0 The [new configuration](https://beta.css.master.co/docs/configuration) may have what you want.

**Comment by 1aron** [comment]
Released at [v2.0.0-beta.143](https://github.com/master-co/css/releases/tag/v2.0.0-beta.143)
